### ISSUE TITLE
Move `runtime-common` to the fellowship

### ISSUE THREAD

**Issue by kianenigma** [polkadot-fellows/runtimes#27]
While creating https://github.com/polkadot-fellows/runtimes/pull/26, it seemed to me that the fact that `runtime-common` is still in `paritytech/polkadot-sdk` is somewhat wrong. This crate contains some critical configurations and I think either entirely, or a subset of it that is highly entangled with the polkadot/kusama runtimes, should be brought into this repo.

**Comment by bkchr** [comment]
Yeah we would need to clean up `runtime-common` as it currently also contains pallets etc.
